### DISCUSSION TITLE
Data without gRNA UMI count matrix

### DISCUSSION THREAD

**Discussion by Larrycpan** [Katsevich-Lab/sceptre#142] [Q&A] [answered]
A very nice tool! I am dealing with perturbation data without gRNA matrix UMI count matrix (low MOI). Is it correct to manually transform the perturbation status of each cell into a binary gRNA matrix for input of the function `import_data`? Additionally, is it possible for SCEPTRE to handle high MOI data where gRNA UMI counts are not available? Thanks for you reply.

**Comment by ekatsevi** [comment]
I will convert this issue to a Discussion and address it there.

**Comment by ekatsevi** [comment, marked as answer]
We are working on a feature to allow you to input gRNA assignment matrices directly. In the meantime, I recommend you input a binary (i.e. 0/1) perturbation assignment matrix to `sceptre` and use `assign_grnas(method = "thresholding", threshold = 1)` to "assign gRNAs". This will have the intended effect. This approach is compatible with both low- and high-MOI data.
